flutter-for-ios.md

```diff
@@ -27,3 +27,1720 @@ that are most relevant to your needs.
+            child: new FlutterLogo(
+              size: 200.0,
+            )),
+          onDoubleTap: () {
+            if (controller.isCompleted) {
+              controller.reverse();
+            } else {
+              controller.forward();
+            }
+          },
+        ),
+      ),
+    );
+  }
+}
+{% endprettify %}
+
+# Theming and text
+
+## How do I theme an app?
+
+You may have noticed a lot of the examples in this document use a `MaterialApp`
+as their app widget. This provides a base of themed widgets to use in your app.
+However these are more themed to an Android device, so in order to use more iOS
+focused widgets you will need to implement these as needed. You can find the
+full set [here](https://flutter.io/widgets/cupertino/).
+
+You could also use a `WidgetApp` as your app widget, which provides some of the
+same functionality, but is not yet as broad as `MaterialApp`.
+
+To customize the colors and styles of any child components you can pass in a
+`ThemeData` object to the `MaterialApp` widget. For example, in the code below
+you can see the primary swatch is set to blue and all text selection color
+should be red.
+
+<!-- skip -->
+{% prettify dart %}
+class SampleApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Sample App',
+      theme: new ThemeData(
+        primarySwatch: Colors.blue,
+        textSelectionColor: Colors.red
+      ),
+      home: new SampleAppPage(),
+    );
+  }
+}
+{% endprettify %}
+
+## How do I set custom fonts on my `Text` widgets?
+
+In iOS, you would import any `ttf` font files into your project and create a
+reference in the `info.plist` file. In Flutter you need to place the font file
+in a folder and reference it in the `pubspec.yaml`, similar to how you import
+images.
+
+<!-- skip -->
+{% prettify yaml %}
+fonts:
+   - family: MyCustomFont
+     fonts:
+       - asset: fonts/MyCustomFont.ttf
+       - style: italic
+{% endprettify %}
+
+Then you would assign the font to your `Text` widget:
+
+<!-- skip -->
+{% prettify dart %}
+@override
+Widget build(BuildContext context) {
+  return new Scaffold(
+    appBar: new AppBar(
+      title: new Text("Sample App"),
+    ),
+    body: new Center(
+      child: new Text(
+        'This is a custom font text',
+        style: new TextStyle(fontFamily: 'MyCustomFont'),
+      ),
+    ),
+  );
+}
+{% endprettify %}
+
+## How do I style my `Text` widgets?
+
+Along with fonts, you can customize other styling elements on a `Text` widget.
+The style parameter of a `Text` widget takes a `TextStyle` object, where you can
+customize many parameters, such as:
+
+- `color`
+- `decoration`
+- `decorationColor`
+- `decorationStyle`
+- `fontFamily`
+- `fontSize`
+- `fontStyle`
+- `fontWeight`
+- `hashCode`
+- `height`
+- `inherit`
+- `letterSpacing`
+- `textBaseline`
+- `wordSpacing`
+
+# Form Input
+
+## What is the equivalent of a placeholder in a text field?
```

Comment:
Kind of abrupt way to introduce forms, directly diving in to placeholders. Shouldnt it mention what widgets are using for forms, and reference the TextEditingController

Reply:
Good point. Added in #993